Tevling/Pages/Statistics.razor.cs

```diff
@@ -151,6 +154,11 @@ protected override async Task OnAfterRenderAsync(bool firstRender)
         await DrawChart();
     }
 
+    private bool IsGarminDeviceSourcedData()
+    {
+        return Devices.Any(d => d.Contains("Garmin"));
```

Comment:
Perhaps this comparison should be case insensitive? Same for the one in `ChallengeService`.
```suggestion
        return Devices.Any(d => d.Contains("Garmin", StringComparison.OrdinalIgnoreCase));
```